src/components/layout/footer.tsx

```
@@ -0,0 +1,54 @@
+          </p>
+        </div>
+
+        <div className="flex flex-col items-center justify-between gap-6 md:flex-row">
+          <div className="flex items-center gap-3 rounded-2xl border border-white/20 px-4 py-2">
+            <span className="flex h-9 w-9 items-center justify-center rounded-full bg-white/20 text-sm font-semibold"></span>
+            <span className="text-sm text-white/80">
+              2024 Electronics Marketplace. All rights reserved.
+            </span>
+          </div>
+          <div className="flex items-center gap-6 text-sm text-white/80">
+            <a href="#" className="transition-colors hover:text-white">
```

Biome (2.3.13)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)